backend/api/postman_collection/models.py

```diff
@@ -0,0 +1,136 @@
+from dataclasses import asdict, dataclass, field
```

Comment:
I think it's better to change this file name to dto.py or some other. I confuced this model.py with django db model.